specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/inputs.json

```diff
@@ -895,46 +950,102 @@
         }
       },
       "x-ms-azure-resource": true
-    }
-  },
-  "parameters": {
-    "SubscriptionIdParameter": {
-      "name": "subscriptionId",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "description": "GUID which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
     },
-    "ResourceGroupNameParameter": {
-      "name": "resourceGroupName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
-      "x-ms-parameter-location": "method"
+    "Compression": {
+      "description": "Describes how input data is compressed",
+      "discriminator": "type",
+      "properties": {
+        "type": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "type"
+      ]
     },
-    "StreamingJobNameParameter": {
-      "name": "jobName",
-      "in": "path",
-      "required": true,
-      "type": "string",
-      "description": "The name of the streaming job.",
-      "x-ms-parameter-location": "method"
+    "Identity": {
+      "description": "Describes how identity is verified",
+      "properties": {
+        "tenantId": {
+          "type": "string"
+        },
+        "principalId": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        }
+      }
     },
+    "azureSqlReferenceInputDataSource": {
```

Comment:
These types appears lack description in properties.

Reply:
Thanks for catching it, I will add descriptions for that.